Formula/clickhouse-odbc.rb

```diff
@@ -0,0 +1,48 @@
+class ClickhouseOdbc < Formula
+  desc "ClickHouse ODBC driver"
+  homepage "https://clickhouse.yandex"
+  url "https://github.com/yandex/clickhouse-odbc.git",
+    :tag => "v1.0.0.20180903",
+    :revision => "17ecca3993e164d2d49360ed9be68becc11b7b05"
+  # head "https://github.com/yandex/clickhouse-odbc.git"
+
+  bottle do
+    # sha256 "256" => :high_sierra
+    # sha256 "256" => :sierra
+    # sha256 "256" => :el_capitan
+  end
+
+  # option "with-debug", "Install debugging version with advanced logs"
+
+  depends_on "cmake" => :build
+  depends_on "libiodbc" => :build
```

Comment:
I am very surprised that libiodbc and openssl are build-only options. Are they not needed at runtime? If you build against these shared libraries, why are they not used at runtime?

Also, where is libtool used?

Reply:
because static .dylib built (to make copyable library, not all users know about brew and want to compile something)
from libtool lib ltdl used via libodbcinst.a

Reply:
The bottling that homebrew does should take care of that shouldn't it ? I don't think we need a statically linked library in here.